src/ci/docker/x86_64-gnu-llvm-8/Dockerfile

```diff
@@ -45,6 +45,8 @@ ENV SCRIPT python2.7 ../x.py test --exclude src/tools/tidy && \
            # (we're only interested in the MIR output, so this doesn't matter)
            python2.7 ../x.py test src/test/mir-opt --pass=build \
                                   --target=armv5te-unknown-linux-gnueabi && \
+           python2.7 ../x.py test src/test/ui --pass=check \
+                                  --target=armv5te-unknown-linux-gnueabi && \
```

Comment:
Before landing, the comment should be adjusted/expanded to also explain why this is here.

Reply:
Good call! Thanks.